packages/documentation-site/patternfly-docs/content/developer-resources/accessibility-design.md

```diff
@@ -0,0 +1,46 @@
+---
+id: Design for accessibility
+section: accessibility
+---
+
+As described in our [accessibility fundamentals](/accessibility/accessibility-fundamentals), users may interact with your product through a variety of assistive technologies. In addition to [developing for accessibility](/accessibility/accessibility-development), you must also incorporate it into your product's UI design. 
+
+## Color and contrast
+
+Color and contrast go hand-in-hand for visual design accessibility. 
+
+For example, you should consider contrast when using colors commonly affected by color perception deficiency (such as those described in [color perception by W3C](https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Overview_of_Low_Vision#Color_Perception)). Different colors in a UI should be distinguishable for users across the color perception deficiency spectrum, typically by ensuring that each color has a different contrast weight. 
+
+Designing with contrast in mind also benefits users with light sensitivity.
+
+[PatternFly's colors](/design-foundations/colors) pass color contrast tests, so long as you follow our recommendations. 
```

Comment:
I'm wondering if we should tweak this line to be a little clearer. In our components, we've tested contrast and are good, but I worry to just say that "PF colors pass color contrast" because you can still use PF colors in a way that does not pass color contrast. That's pretty much true with any color palette, for example [slide 40 of this a11y design presentation](https://docs.google.com/presentation/d/1xomLQRUJ2IJR4n-tRyK13B09G2IkcMGXy1p8_OapDcI/edit#slide=id.ge5933efd32_1_3392) shows the RH color palette with the different combinations of colors that wouldn't be accessible. Maybe we can say something like, "[PatternFly's colors](/design-foundations/colors) pass color contrast tests in our components, so long as you follow our recommendations. If you combine PF colors in other ways, please check the color contrast with a contrast checker"?